web/genericl10n.js

```diff
@@ -15,14 +15,33 @@
 
 /** @typedef {import("./interfaces").IL10n} IL10n */
 
+import { FeatureTest, fetchData } from "pdfjs-lib";
 import { FluentBundle, FluentResource } from "fluent-bundle";
 import { DOMLocalization } from "fluent-dom";
-import { fetchData } from "pdfjs-lib";
 import { L10n } from "./l10n.js";
 
+function PLATFORM() {
+  const { isAndroid, isMac, isLinux, isWindows } = FeatureTest.platform;
```

Comment:
nit: `const { platform: { ... }} = FeatureTest;`

Reply:
Hmm, I'm not sure if that really improves readability so I'll leave it as-is.